### DISCUSSION TITLE
Getting weird error in a set of users machines: Error Code: Aborted - Error: ERR_ABORTED

### DISCUSSION THREAD

**Discussion by SenthilRG** [cefsharp/CefSharp#4695] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

114.2.100

### Operating System

Windows 10

### Architecture

AnyCPU

### .Net Version

.Net 4.7.2

### Implementation

WPF

### Reproduction Steps

I have created WPF based application and installed CefSharp WPF NuGet library (v114.2.100) in my application. I am trying to load Authorization URL inside Chromium browser. There are some page redirections happening while initiating the authorization process. I am listening those page load process by using LoadingStateChanged event. While loading the specific URL, it's failing to load in a set of users machines. But it's working fine in rest of the users machines including my machine as well. The exception has been captured under LoadError event and the error which we are getting is "Error Code: Aborted - Error: ERR_ABORTED". 

**Code:**

**App.xaml.cs:**

```
#if ANYCPU
        //Only required for PlatformTarget of AnyCPU
        CefRuntime.SubscribeAnyCpuAssemblyResolver();
    #endif
    var settings = new CefSettings()
    {
        //By default CefSharp will use an in-memory cache, you need to specify a Cache Folder to persist data
        CachePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "CefSharp\\Cache")
    };
    settings.CefCommandLineArgs.Add("enable-media-stream");
    settings.CefCommandLineArgs.Add("use-fake-ui-for-media-stream");
    settings.CefCommandLineArgs.Add("enable-usermedia-screen-capturing");
    if (!Cef.IsInitialized)
    {
        var t = Cef.InitializeAsync(settings, performDependencyCheck: true, browserProcessHandler: null);
        t.Wait();
    }
```

**AuthorizationView.xaml.cs:**

```
ChromiumWebBrowser browser = new ChromiumWebBrowser(url);
    grd.Children.Add(browser);
    browser.LoadingStateChanged += (sender, args) =>
    {
        if (args.IsLoading == false)
        {
            this.Dispatcher.Invoke(() =>
            {
              //code
            }
        }
    };
    browser.LoadError += (sender, args) =>
    {
        this.Dispatcher.Invoke(async() =>
        {
            //Error code
        }
    };
```

### Expected behavior

The URL should be loaded without any issue and should be captured under LoadingStateChanged event in all the users machines.

### Actual behavior

The URL is failing to load in a set of users machines and the error has been captured LoadError event.

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by SenthilRG** [comment]
@amaitland
I am not able to reproduce this issue in my local machine. Some of the users are facing this issue. Please help me to resolve this issue.

**Reply by amaitland** [reply]
> CefSharp Version
> 114.2.100

Please don't open issues for old unsupported versions. Use discussions to ask support questions. 

> The URL is failing to load in a set of users machines and the error has been captured LoadError event.

Did you check the log file for additional information? https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

What troubleshooting have you done? What's different between the machines that work and ones that don't?

This sort of issue is often environmental, security software, firewall, etc.

**Reply by SenthilRG** [reply]
@amaitland 
I have upgraded CefSharp.Wpf Nuget package with 120.2.70 version and added below piece of code to log the error in the local machine.

```
var settings = new CefSettings()
{
    //By default CefSharp will use an in-memory cache, you need to specify a Cache Folder to persist data
    LogFile = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "CefSharp\\Debug.log"),
    LogSeverity = LogSeverity.Error,
    CachePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "CefSharp\\Cache")
};
```

Also I have tried the below piece of code as well.

```
var settings = new CefSettings();
settings.CefCommandLineArgs.Add("disable-gpu"); // Disable GPU acceleration
settings.CefCommandLineArgs.Add("disable-gpu-vsync"); //Disable GPU vsync
```

When I try to debug the application, I am getting the below error log from the user's machine. In my machine, the error log file is empty because I am not facing any issues.

> [0129/133158.109:ERROR:shared_image_manager.cc(217)] SharedImageManager::ProduceSkia: Trying to Produce a Skia representation from a non-existent mailbox.

chrome://version/ - Not Working Machine Output:

![image](https://github.com/cefsharp/CefSharp/assets/30817145/c47c3a70-b074-4f8c-8fa2-4b074927e398)

chrome://gpu/ - Not Working Machine Output:

![image](https://github.com/cefsharp/CefSharp/assets/30817145/f6baaeaf-9edb-48a5-903c-df1713ee6c95)

chrome://version/ - Working Machine Output:

![image](https://github.com/cefsharp/CefSharp/assets/30817145/e2f31e6c-df40-482d-aff0-3f7b09c5b141)

chrome://gpu/ - Working Machine Output:

![image](https://github.com/cefsharp/CefSharp/assets/30817145/16db5cd5-1c8d-41fe-be4d-c57d1a44517d)

**Reply by SenthilRG** [reply]
@amaitland 
I need your help to resolve this issue.

```
[0130/141425.837:WARNING:frame_impl.cc(420)] SendProcessMessage sent to detached frame 30064771073 [7,1] will be ignored
[0130/141425.994:ERROR:shared_image_manager.cc(220)] SharedImageManager::ProduceSkia: Trying to Produce a Skia representation from a non-existent mailbox.
```

**Comment by SenthilRG** [comment]
@amaitland,
I am struggling with this issue for more than 1 week. Please throw some insights to resolve this issue.

**Comment by SenthilRG** [comment]
@amaitland
I am waiting for your reply.

**Comment by SenthilRG** [comment]
N/A
